src/components/victory-chart/victory-chart.js

```diff
@@ -307,6 +315,7 @@ export default class VictoryChart extends React.Component {
         ref: index,
         key: index,
         standalone: false,
+        theme: props.theme,
```

Comment:
hmm @boygirl removing this actually made victorychart stop passing down theme to child elements